wolfssl/wolfio.h

```diff
@@ -406,9 +406,17 @@ WOLFSSL_API void wolfSSL_SetIOWriteFlags(WOLFSSL* ssl, int flags);
 
 #ifndef XINET_NTOP
     #define XINET_NTOP(a,b,c,d) inet_ntop((a),(b),(c),(d))
+    #ifdef _WIN32 /* Windows-friendly definition */
```

Comment:
Typically we use `#ifdef USE_WINDOWS_API` for better compatibility (see settings.h:466).